Formula/gibo.rb

```diff
@@ -16,11 +19,13 @@ class Gibo < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "010d9cc6f4ec371787d333657483eeb7980b640041688a60514db8376e44eaa8"
   end
 
+  depends_on "go" => :build
+
   def install
-    bin.install "gibo"
-    bash_completion.install "shell-completions/gibo-completion.bash"
-    zsh_completion.install "shell-completions/gibo-completion.zsh" => "_gibo"
-    fish_completion.install "shell-completions/gibo.fish"
+    date = DateTime.now.iso8601
+    ldflags = "-s -w -X github.com/simonwhitaker/gibo/cmd.version=#{version} -X github.com/simonwhitaker/gibo/cmd.date=#{date}"
```

Comment:
you can just reference #{time.iso8601}, there are many examples in the repo

Reply:
Oh nice! Thanks, will update.

Reply:
Sounds great. :)